docs/demo/deployments/xcluster/ovl/meridio/README.md

```diff
@@ -31,3 +31,24 @@ issues, where usually one of the VPP forwarder gets killed by the OS.
 
 Refer to [Meridio description](https://github.com/Nordix/Meridio/blob/master/docs/demo/xcluster.md) for details. 
 
+
+## Private registry and local manifests
+
+Generate local manifests in the "default/" directory and check images
+missing in the `private-reg`;
+
+```
+mdir=$(readlink -f ../../../../../..)
+dst=./default/etc/kubernetes/meridio
+mkdir -p $dst
+for n in spire nsm nsm-vlan; do
```

Comment:
nsm is not used anymore, nsm-vlan is the one to go with
(at least until vlan forwarder functionality gets merged into vpp)